clients/admin-ui/src/features/system/SystemCard.tsx

```diff
@@ -57,7 +57,7 @@ const SystemCard = ({ system }: SystemCardProps) => {
 
   return (
     <Box display="flex" data-testid={`system-${system.fides_key}`}>
-      <Box flexGrow={1} p={4}>
+      <Box flexGrow={1} p={4} as="button" onClick={handleEdit}>
```

Comment:
this works, but technically isn't semantically correct: https://stackoverflow.com/questions/12982269/is-it-semantically-incorrect-to-put-a-div-or-span-inside-of-a-button

that's not the biggest deal, but it also changes the rendering a bit:
| Before | After |
|-------|-------|
| <img width="415" alt="image" src="https://github.com/ethyca/fides/assets/24641006/4113a434-29e3-4757-859c-46a97736aa77">| <img width="403" alt="image" src="https://github.com/ethyca/fides/assets/24641006/602c5f67-c7f4-4fff-a7ac-bfb6a838c1d4">|

I think a way that could solve both problems is, instead of using a button component, wrap the box in a `NextLink`. We do a similar thing in the home page: https://github.com/ethyca/fides/blob/f526d9ffb176006d701493c9d0eff6b4884e811f/clients/admin-ui/src/home/HomeContent.tsx#L33

Incidentally, I think [this line](https://github.com/ethyca/fides/blob/f526d9ffb176006d701493c9d0eff6b4884e811f/clients/admin-ui/src/features/system/SystemCard.tsx#L41) may be redundant now, since we also set the active system in the configure page:
https://github.com/ethyca/fides/blob/f526d9ffb176006d701493c9d0eff6b4884e811f/clients/admin-ui/src/pages/systems/configure/%5Bid%5D.tsx#L37-L39
so you don't necessarily have to rely on `handleEdit`, it should be enough to just pass the route to a `NextLink`